app/HutchManager/Data/Migrations/ApplicationDbContextModelSnapshot.cs

```diff
@@ -181,7 +181,7 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.Property<string>("MessageTemplate")
                         .IsRequired()
                         .HasColumnType("text")
-                        .HasColumnName("messagetemplate");
+                        .HasColumnName("message_template");
```

Comment:
From the manager this is "messagetemplate" not "message_template"

Reply:
it's been changed in the Entity too though (hence the migration) so the manager itself will create it with `message_template`. Not sure how Serilog feels about that though ;)

Reply:
I changed the name of them in both the agent and manager to "message_template". In the manager I just the bit above in the square brackets. All the c# code will still use `MessageTemplate`

Reply:
yeah not sure if Serilog would use message_template

Reply:
It might be easier to keep messagetemplate, so Serilog doesn't complain for now? Unless we have a reason we want it as message_template?